main.tf

```diff
@@ -137,3 +290,32 @@ resource "observe_link" "ingress_logs" {
     }, var.link_targets
   )
 }
+
+resource "observe_board" "nginx_ingress" {
+  count   = var.enable_nginx_ingress_metrics ? 1 : 0
+  dataset = var.kubernetes.ingress.oid
+  name    = "Monitoring"
```

Comment:
This should be NGINX specific name. You may install another ingress module for, say, haproxy.

Reply:
Changed.  Leaving comment unresolved to make sure the change was appropriate.